stored_requests/backends/db_fetcher/fetcher_test.go

```diff
@@ -52,6 +57,24 @@ func TestGoodResponse(t *testing.T) {
 	assertHasData(t, storedImps, "imp-id-2", `{"imp":true,"value":2}`)
 }
 
+func TestGoodResponseFetchResponses(t *testing.T) {
+	mockQuery := "SELECT id, data, 'response' AS dataType FROM responses_table WHERE id IN (?, ?)"
+	mockReturn := sqlmock.NewRows([]string{"id", "data", "dataType"}).
+		AddRow("resp-id-1", `{"resp":false,"value":1}`, "response").
+		AddRow("resp-id-2", `{"resp":true,"value":2}`, "response")
+
+	mock, fetcher := newFetcher(t, mockReturn, mockQuery, "resp-id-1", "resp-id-2")
+	defer fetcher.db.Close()
+
+	storedResponses, errs := fetcher.FetchResponses(context.Background(), []string{"resp-id-1", "resp-id-2"})
+
+	assertMockExpectations(t, mock)
+	assertErrorCount(t, 0, errs)
+	assertMapLength(t, 2, storedResponses)
+	assertHasData(t, storedResponses, "resp-id-1", `{"resp":false,"value":1}`)
+	assertHasData(t, storedResponses, "resp-id-2", `{"resp":true,"value":2}`)
+}
+
```

Comment:
Similar comment to above, where I think the TestGoodRepsonseFetchResponses, TestPartialResponse, and TestEmptyResponse tests could be consolidated into one table driven test.

Reply:
Done